tests/checkbox/checkbox-provider-tdx/units/tests/jobs.pxu

```diff
@@ -83,20 +83,20 @@ command:
   export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/:$LD_LIBRARY_PATH
   setup-env-and-run test_boot_multiple_vms.py
 
-id: tdx-basic/td-stress-boot
-category_id: tdx-basic
-flags: simple
-_summary: Test boot stress
-depends:
-after:
-requires:
-  executable.name == 'qemu-system-x86_64'
-command:
-  export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/:$LD_LIBRARY_PATH
-  setup-env-and-run test_stress_boot.py
+#id: tdx-stress/td-stress-boot
```

Comment:
Did you temporarily comment this out and forget to remove?

Reply:
The td-stress-boot takes a VERY long to complete.  So I commented it out for now and was planning on going back to it and maybe better categorizing it.  Some tests fail too so there is more work to do.  This is an attempt to get the framework.